src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -68,10 +68,11 @@ class ViewClippingWindow final : public Window
 {
 private:
     CoordsXY _selectionStart;
-    CoordsXY _previousClipSelectionA;
-    CoordsXY _previousClipSelectionB;
+    CoordsXY _previousClipSelectionA = { 0, 0 };
+    CoordsXY _previousClipSelectionB = gMapSize.ToCoordsXY();
     bool _toolActive{ false };
     bool _dragging{ false };
+    bool _firstStart = true;
```

Comment:
This gets called when the window is constructed, not on park load, so with this change the clip-selection will reset every time the window is opened, not only once per map.

Reply:
This was removed